README.md

```diff
@@ -1,96 +1,50 @@
 # Operator Mono Ligatures
-
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
 
-<img src="./images/operator-mono-lig.png" />
-
-This project will generate new OpenType fonts for [Operator Mono](https://www.typography.com/fonts/operator/styles/) that includes ligatures similar to
-those found in the popular [Fira Code](https://github.com/tonsky/FiraCode) font.
-
+This project will generate new OpenType fonts for [Operator Mono](https://www.typography.com/fonts/operator/styles/) that includes ligatures similar to those found in the popular [Fira Code](https://github.com/tonsky/FiraCode) font.
 These ligatures were custom created using [Glyphs](https://glyphsapp.com/).
```

Comment:
@mskelton Is it worth changing this to specifically [Glyphs Mini](https://glyphsapp.com/glyphs-mini) instead of just Glyphs?

Reply:
I agree, as the full Gylphs app added info into the glyphs file that is not processed by Mini and may cause merge issues. I'd rather have everyone work with the same tools if possible.

The URL for Mini is https://glyphsapp.com/glyphs-mini